hypervisor/arch/x86/configs/vm_config.c

```diff
@@ -19,6 +19,7 @@
 		.name = VM##idx##_CONFIG_NAME,	\
 		.pcpu_bitmap = VM##idx##_CONFIG_PCPU_BITMAP,	\
 		.guest_flags = VM##idx##_CONFIG_FLAGS,	\
+		.clos = VM##idx##_CONFIG_CLOS,	\
```

Comment:
I prefer to directly use the number here, rather than the complex MACRO...

Reply:
clos is configurable for each scenario, so I use MACRO here.